source/PvL_07.rst

```diff
@@ -330,22 +274,29 @@ We'll need to get a few things right before this works:
    (as some students are), this may require some more special handling to work around our proxy.  
    Use a local resource for the purpose of this demonstration! 
   
-Here is a slightly different example.  Rather than save the web resource to
-our local disk, we read it directly into a string, and return it:
+Here is a slightly different example using the *requests* module. This module is not part of the standard library distributed with python, however it is easier to use and significantly more potent than the *urllib* module distributed with python. Read *requests* documentation on `<http://docs.python-requests.org/>`_ to learn how to install and use the module. Here, rather than save the web resource to
+our local disk, we read it directly into a string, and we print that string:
 
-    .. sourcecode:: python3
-        :linenos:
-        
-        import requests
+.. sourcecode:: python3
+    :linenos:
+    
+    import requests
 
+    url = "http://xml.resource.org/public/rfc/txt/rfc793.txt"
+    response = requests.get(url)
+    print(response.text)
 
-        the_text = requests.get("http://xml.resource.org/public/rfc/txt/rfc793.txt")
-        print(the_text)
-        
-Opening the remote url returns what we call a **socket**.  This is a handle to 
-our end of the connection between 
-our program and the remote web server.  We can call read, write, and close methods on
-the socket object in much the same way as we can work with a file handle.
+Opening the remote URL returns the response from the server. That response contains several informations, and the *requests* module allows us to access them in various ways. On line 5, we get the downloaded document as a single string. We could also read it line by line as follows:
```

Comment:
"... contains several informations..." is not English. "... contains several *types of* information..." or "... contains a lot of information (besides the text)...", ...?